bigquery/snippets/table/bigquery_create_table_from_template.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 Google LLC
```

Comment:
Stop living in the past. :-)

```suggestion
// Copyright 2021 Google LLC
```

Reply:
Copy/paste error or an homage to pre-pandemic times?  I'll never tell.